app/models/manageiq/providers/cloud_manager/vm.rb

```diff
@@ -25,6 +26,8 @@ class ManageIQ::Providers::CloudManager::Vm < ::Vm
   virtual_column :ipaddresses,   :type => :string_set, :uses => {:network_ports => :ipaddresses}
   virtual_column :mac_addresses, :type => :string_set, :uses => :network_ports
 
+  supports_not :resize, :reason => _("Resize Operation is not available for VM.")
```

Comment:
I think this should be in VmOrTemplate or in Vm

@Fryguy should we move everything to VmOrTemplate or are there some things that should be either in Vm or in Template